lms/static/js/views/fields.js

```diff
@@ -137,10 +164,8 @@
                 if (xhr.status === 400) {
                     try {
                         var errors = JSON.parse(xhr.responseText),
-                            validationErrorMessage = _.escape(
-                                errors.field_errors[this.options.valueAttribute].user_message
-                            ),
-                            message = this.indicators.validationError + validationErrorMessage;
+                            validationErrorMessage = errors.field_errors[this.options.valueAttribute].user_message,
+                            message = HtmlUtils.HTML(this.indicators.validationError + validationErrorMessage);
```

Comment:
1. Rename message to messageHtml, or is this one that might be either?
2. This would be another nice place to use `HtmlUtils.joinHtml()`. :)